### ISSUE TITLE
Slack Interactive Messages

### ISSUE THREAD

**Issue by pproenca** [ramswaroop/jbot#25]
Hey,

Is there any plans to integrate the new Slack API interactive messages feature?

https://api.slack.com/docs/message-buttons

**Comment by ramswaroop** [comment]
Hey,

Currently, JBot supports Slack's RTM API, Webhooks and Slash commands and not web api. While working on the web api I will definitely include message button feature. It's just a POST call after all similar to web apis. You can do it one line in JBot (which uses spring-boot).

**Comment by pproenca** [comment]
How would that be?

I guess the only thing it makes it more complicated is the requirement to register a `Slack app` 😢

**Comment by ramswaroop** [comment]
@pproenca Yes, I agree. You have to make a Slack app and your message action receiving url should be `https`. But why is it complicated? You just have to host your jbot in some server and release it to Slack app store as an app.

**Comment by pproenca** [comment]
Isn't the Oauth2 required as well?

**Comment by Swapnil2004** [comment]
@rampatra is the above issue closed?
